### ISSUE TITLE
tv64 not a structure or a union member

### ISSUE THREAD

**Issue by ghibo** [hhoffstaette/kernel-patches#13]
With latest kernel 4.9.40, the compilation stops with an with error that .tv64 (of file kernel/alarm/timer.c) is not a structure nor a union member. The following code (ktime-20170728-003-get-rid-of-the-union.patch) should complete the previous ktime patchset:

```
--- linux-4.9/kernel/time/alarmtimer.c.tv64     2017-07-28 09:00:27.782593329 +0200
+++ linux-4.9/kernel/time/alarmtimer.c  2017-07-28 09:04:24.868957185 +0200
@@ -662,7 +662,7 @@ static int alarm_timer_set(struct k_itim
         * Rate limit to the tick as a hot fix to prevent DOS. Will be
         * mopped up later.
         */
-       if (timr->it.alarm.interval.tv64 &&
+       if (timr->it.alarm.interval &&
                        ktime_to_ns(timr->it.alarm.interval) < TICK_NSEC)
                timr->it.alarm.interval = ktime_set(0, TICK_NSEC);

```

**Comment by hhoffstaette** [comment]
This was fixed already (in 265e276427) but simply not pushed yet, since Greg released this kernel when I was asleep. It's up now.

**Comment by ghibo** [comment]
Great!
